constants.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    graphAPIEndpoints: {
+        masterchef: 'https://api.thegraph.com/subgraphs/name/sushiswap/master-chef',
+        bar: 'https://api.thegraph.com/subgraphs/name/sushiswap/sushi-bar',
+        timelock: 'https://api.thegraph.com/subgraphs/name/sushiswap/sushi-timelock',
+        maker: 'https://api.thegraph.com/subgraphs/name/sushiswap/sushi-maker',
+        exchange: 'https://api.thegraph.com/subgraphs/name/sushiswap/exchange',
+        exchangev1: 'https://api.thegraph.com/subgraphs/name/jiro-ono/sushiswap-v1-exchange',
```

Comment:
Idk the official javascript syntax for something like this, but I feel like `exchange_v1` looks better than `exchangev1`.

Though this will involve having to change it it in multiple places, so tell me if you think this is a useless change.

Reply:
IMO it doesn't really matter, but you're right, it looks better. A simple replace all will do the trick